pmm/pmm2-server-release.groovy

```diff
@@ -171,24 +171,35 @@ pipeline {
                     DOCKER_MID="\$TOP_VER.\$MID_VER"
                     sg docker -c "
                         set -ex
+                        # push pmm-server
                         docker pull \${DOCKER_VERSION}
+                        docker tag \${DOCKER_VERSION} percona/pmm-server:latest
+                        docker push percona/pmm-server:latest
+                        docker tag \${DOCKER_VERSION} perconalab/pmm-server:latest
```

Comment:
i thought that we intentionally set `dev-latest` for `perconalab` to not confuse our users.

Reply:
it's in the different pipeline